fuzzing/scripts/build/bzip2.sh

```diff
@@ -26,6 +26,9 @@ if [[ "${#}" == "0" || "${1}" != "--no-init" ]]; then
     git rev-parse HEAD
 fi
 
-make -j$( nproc )
+if [[ -f "${path_to_src}/Makefile" ]]; then
+  cd ${path_to_src}
```

Comment:
After rebasing this change, this is actually now the real fix, since previously a there wasn't always a directory change to the source.